certificate-service/app/src/main/resources/fk7210/schematron/igrav.v1.sch

```diff
@@ -14,31 +14,31 @@
 
   <iso:pattern id="intyg">
     <iso:rule context="//rg:intyg">
-      <iso:assert test="count(gn:svar[@id='1']) = 1">
-        Ett intyg måsta ha ett 'Beräknat nedkomstdatum' svar
+      <iso:assert test="count(gn:svar[@id='54']) = 1">
+        Ett intyg måsta ha ett 'Beräknat födelsedatum' svar
       </iso:assert>
     </iso:rule>
   </iso:pattern>
 
   <iso:pattern id="q1">
     <iso:rule context="//gn:svar[@id='1']">
```

Comment:
Should this be updated to 54 aswell?

Reply:
Nice catch, was surprised that the tests didn't catch this though. Fixed!